src/Sylius/Bundle/CurrencyBundle/Resources/config/serializer/Model.Currency.yml

```diff
@@ -19,14 +19,6 @@ Sylius\Component\Currency\Model\Currency:
             expose: true
             type: boolean
             groups: [Default, Detailed]
-        createdAt:
-            expose: true
-            type: DateTime
-            groups: [Detailed]
-        updatedAt:
```

Comment:
why? It would come in handy to know when was the exchange rate modified. Or am i missing something